packages/react-components/src/organisms/TeamCreateOutputForm.tsx

```diff
@@ -100,7 +100,7 @@ const TeamCreateOutputForm: React.FC<TeamCreateOutputFormProps> = ({
       onSave={() =>
         onSave({
           tags,
-          link,
+          link: link === '' ? undefined : link,
```

Comment:
String(link).trim()

Reply:
Yeah. To be sure, you mean `String(link).trim() === ''`, right?